File: tests/fixtures/languages/Sample.cs
namespace Acme.Tools;

public interface IReader
{
    string Name { get; }
    void Read();
    void Close() { }
}

public class Service
{
    private string name = "", alias = "";
    public string Title { get; set; }
    public Service() { }
    public void Render()
    {
        void Hidden() { }
        class Local { }
    }
    public abstract void Missing();
    public class Inner
    {
        public void Save() => Console.WriteLine("saved");
    }
}

public enum State
{
    Ready,
    Named = 2,
}

public delegate void Handler(string value);

public class Café
{
    public string Crème;
}

File: tests/fixtures/languages/Sample.java
package fixtures;

class Service {
    String name, alias;

    Service() {}

    void render() {
        int local = 1;
        class Hidden {}
    }

    private void save() {}

    class Inner {
        void render() {}
    }
}

interface Reader {
    String read();

    default void close() {}
}

enum State {
    READY,
    NAMED(2);

    State() {}
    State(int value) {}
}

record Result<T>(T value) {
    T unwrap() { return value; }
}

@interface Marker {
    String value();
}

class Café {
    String crème;
}
